### DISCUSSION TITLE
Node 17

### DISCUSSION THREAD

**Discussion by kaszperro** [RobinTail/express-zod-api#860] [Q&A] [answered]
Why node 17 is not supported in version 8.x? Is it possible to support it?

**Comment by RobinTail** [comment, marked as answer]
Because of this, @kaszperro : 
https://github.com/RobinTail/express-zod-api/pull/569

Jest 29 does not support it:
https://github.com/facebook/jest/pull/13033

The reason is EOL (end of life).

**Reply by RobinTail** [reply]
> Is it possible to support it?

### Short answer

No ¯\\\_(ツ)_/¯ 

### Long answer

Node 17 end of life was 9 months ago (01 Jun 2022).

> Major Node.js versions enter Current release status for six months, which gives library authors time to add support for them. After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to Active LTS status and are ready for general use. LTS release status is “long-term support”, which typically guarantees that critical bugs will be fixed for a total of 30 months.

> **Production applications should only use Active LTS or Maintenance LTS releases.**
> https://endoflife.date/nodejs

Sorry, @kaszperro 

Consider using `nvm` for Node version control.
If you're having troubles running your project on Node 18, try upgrading your dependencies step-by-step.
